post-processing/clickhouseImport.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-process-env */
+const fs = require('fs').promises;
+const path = require('path');
+const Clickhouse = require('../reporters/ClickhouseReporter');
+
+const ch = new Clickhouse();
+const crawlName = process.env.CRAWL_NAME || '';
+const crawlRegion = process.env.REGION || '';
+const crawledPagePath = process.argv[2];
+console.log(crawledPagePath);
```

Comment:
can you please clean up console.log's from this file? It's weird that eslint is not complaining about it 🤔